tests/cpu/vu0_macro/transfer.expected

```diff
@@ -0,0 +1,35 @@
+-- TEST BEGIN
+lqc2:
+  lqc2 +0: c0dec0de beefdead abcdef01 23456789
+  lqc2 +16: 44556677 00112233 ccddeeff 8899aabb
+  lqc2 -16: c0de1337 deadbeef 9abcdef0 45678123
+  lqc2 -> $vf0: 3f800000 00000000 00000000 00000000
```

Comment:
Yeah, VF0 is hardwired to (0.0f, 0.0f, 0.0f, 1.0f), just checking that LQC2 doesn't change its value. Maybe I should change the value displayed for something with a better meaning?